torch/_inductor/scheduler.py

```diff
@@ -37,7 +37,7 @@
 from torch.utils._sympy.symbol import free_symbol_is_type, SymT
 from torch.utils._triton import has_triton
 
-from . import comms, config, dependencies, ir, metrics
+from . import comms, config, dependencies, ir, metrics, spmd_prefetch, spmd_bucketing
```

Comment:
IMO there's no need to have two separate files, we can put them into one file and maybe call it `simple_fsdp`